bigquery_etl/glam/templates/histogram_bucket_counts_v1.sql

```diff
@@ -23,11 +25,12 @@ build_ids AS (
       COUNT(DISTINCT client_id) > {{ minimum_client_count }}),
 normalized_histograms AS (
   SELECT
+    sampled,
     {{ attributes }},
     ARRAY(
       SELECT AS STRUCT
         {{metric_attributes}},
-        mozfun.glam.histogram_normalized_sum(value, 1.0) AS aggregates
+        mozfun.glam.histogram_normalized_sum(value, IF(sampled, 10.0, 1.0)) AS aggregates
```

Comment:
It seems like for histogram bucket counts the `os` and `channel` columns are included in the attributes list, and therefore they'd be available to use here, so could you just do something like this rather than using the SQL injection approach?
```suggestion
        mozfun.glam.histogram_normalized_sum(
          value,
          IF(all_combos.os = 'Windows' AND all_combos.channel = 'release', 10.0, 1.0)
        ) AS aggregates
```